templates/meta_review.html

```diff
@@ -1,53 +1,145 @@
-                <p>give away: </p>
-                <p>number of positive reactions: {{ participant.pos_out }}</p>
-                <p>number of negative reactions: {{ participant.neg_out }}</p>
-                <p>
-                  negative point offset:
-                  {{ participant.offset|floatformat:2 }}
-                </p>
-                <p>weight_factor: {{ participant.weight_factor|floatformat:2 }}</p>
-              </div>
+{% extends 'base.html' %}
+{% load staticfiles %}
+{% block title %}
+  Community | Meta-Review
+{% endblock %}
+
+{% block add_css_files %}
+  <link rel="stylesheet" href="{% static 'css/contributors.css' %}">
+  <link rel="stylesheet" href="{% static 'css/meta-reviewers.css' %}">
+{% endblock %}
+
+{% block add_js_files %}
+  <script src="{% static 'js/contributors.js' %}"></script>
+{% endblock %}
+
+{% block main-content %}
+
+  <div class="web-page-details apply-flex center-content">
+    <h3 style="padding-right: 15px">~</h3>
+    <h3 class="page-name">
+      Contributors Meta-Review Details
+    </h3>
+    <h3 style="padding-left: 15px">~</h3>
+  </div>
+
+  <div class="apply-flex center-content">
+    <p class="container web-page-description">
+      Contributor meta-review related details are calculated based upon the
+      GitHub reactions made by that contributor on any issue or merge request
+      within an organization, for example - {{ org.name }}.
+    </p>
+  </div>
+
+  <div class="contributors-section apply-flex center-content">
+    <div class="form-fields">
+      <form>
+        <div class="input-field apply-flex center-content search-field">
+          <i class="fa fa-search social-icons"></i>
+          <input id="search" type="search" autocomplete="off" placeholder="Search by username or name" required>
+          <i class="fa fa-close social-icons"></i>
+        </div>
+      </form>
+    <div class="apply-flex center-content">
+    </div>
+      <div class="search-results">
+        <table>
+          <thead>
+            <tr>
+              <th>Search Results</th>
+            </tr>
+          </thead>
+          <tbody class="search-results-tbody large-font">
+            <tr>
+              <td>
+                No results found!
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+
+  <div class="contributors-cards">
+  {% for contributor in contributors_meta_review_details %}
+  <div class="contributor-card meta-reviewer apply-flex" login="{{ contributor.login }}" name="{{ contributor.name }}">
+    <div class="contributor-image meta-reviewer-image">
+      <img src="//github.com/{{ contributor.login }}.png/?size=210"
+           alt="user-image">
+      <div class="top-right large-font bold-text">
+        {{ contributor.rank }}
+      </div>
+    </div>
+    <div class="metareviewer-details">
+      <div class="contributor-details-part-1 column-flex">
+        <a class="bold-text large-font" href="//github.com
```

Comment:
Link is not working for me

Reply:
I will check if it is feasible to make it case insensitive!
And will also check why the link the isn't working as it is an anchor tag

Reply:
Made search case insensitive and the link too now is working :+1: